modules/auth/user_form.go

```diff
@@ -79,7 +79,7 @@ func (f *InstallForm) Validate(ctx *macaron.Context, errs binding.Errors) bindin
 type RegisterForm struct {
 	UserName           string `binding:"Required;AlphaDashDot;MaxSize(40)"`
 	Email              string `binding:"Required;Email;MaxSize(254)"`
-	Password           string `binding:"Required;MaxSize(255)"`
+	Password           string `binding:"MaxSize(255)"`
```

Comment:
Does this change need a migration?

Reply:
I'm pretty sure no, because this is a form validation not a struct that xorm uses.

Reply:
ah sorry of course!